workshop/SCHEDULE.md

```diff
@@ -3,15 +3,44 @@ title: Virtual Workshop Schedule
 nav_title: Schedule
 ---
 
-<!--See an example from a past virtual workshop here: https://github.com/AlexsLemonade/2020-may-training/wiki/Schedule --> 
+A preliminary schedule for the September 2022 Data Lab Virtual Training Workshop appears below.
 
-| Time        | Topic                                          |
-|-------------|------------------------------------------------|
-| **Day 1**   | **Date** <br> [Module]()                      |
-| 12:00 PM    | Welcome, Introductions and Getting Started     <br>[Welcome Slides (PDF)](../slides/ Workshop_Introduction.pdf)|
-| 5:00        | End             |
-| **Day 2**   | **Date**  | 
-| **Day 3**   | **Date**  |               
-| **Day 4**   | **Date**  | 
-| **Day 5**   | **Date**  |     
-| 5:00        | Adjourn   |
+
+*Note: All times are [EDT (UTC−04:00)](https://www.timeanddate.com/time/zones/edt).*
+
+
+| Time        | Topic                             | Location |
+|-------------|--------------------------------------------|----------------|
+| **Day 1**   | **2022-09-19** <br> [_**Introduction to R and the Tidyverse**_](https://github.com/AlexsLemonade/training-modules/tree/master/intro-to-R-tidyverse)                      |
```

Comment:
We could probably eliminate these header links too for now, but either way should be fine.
```suggestion
| **Day 1**   | **2022-09-19** <br> _**Introduction to R and the Tidyverse**_                 |
```